docs/ref/settings.txt

```diff
@@ -1669,6 +1669,9 @@ Default: ``'``:class:`django.forms.renderers.DjangoTemplates`\ ``'``
 The class that renders form widgets. It must implement :ref:`the low-level
 render API <low-level-widget-render-api>`.
 
+In addition to the default renderer, Django also includes a Built-in-template
+form renderer for Jinja2 ``'``:class:`django.forms.renderers.Jinja2`\ ``'``.
```

Comment:
Maybe we can use a list to be consistent with other setting docs, e.g.
```diff
diff --git a/docs/ref/settings.txt b/docs/ref/settings.txt
index 12d89142d1..d145a70271 100644
--- a/docs/ref/settings.txt
+++ b/docs/ref/settings.txt
@@ -1667,7 +1667,10 @@ generate correct URLs when ``SCRIPT_NAME`` is not ``/``.
 Default: ``'``:class:`django.forms.renderers.DjangoTemplates`\ ``'``
 
 The class that renders form widgets. It must implement :ref:`the low-level
-render API <low-level-widget-render-api>`.
+render API <low-level-widget-render-api>`. Included form renderers are:
+
+* ``'``:class:`django.forms.renderers.DjangoTemplates`\ ``'``
+* ``'``:class:`django.forms.renderers.Jinja2`\ ``'``
 
 .. setting:: FORMAT_MODULE_PATH
 
```

Reply:
@smithdc1 I can push edits if you agreed with my suggestion. :point_up:

Reply:
Yes. This makes sense. Thanks for the help here. 👍